docs/kyma/docs/04-02-local-installation.md

```diff
@@ -20,140 +18,132 @@ Additionally, download these tools:
 
 Virtualization:
 
-- [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - Mac only
+- [Hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver) - MacOS only
 - [VirtualBox](https://www.virtualbox.org/) - Linux only
 
-> **NOTE:** To work with Kyma, use only the provided scripts and commands. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command.
+> **NOTE:** To work with Kyma, use only the provided scripts and commands. Kyma requires a specific Minikube configuration and does not work on a basic Minikube cluster that you can start using the `minikube start` command.
+
+## Install Kyma and access the Console UI
 
-## Set up certificates
+1. Open a terminal window and navigate to a space in which you want to store your local Kyma sources.
 
-Kyma comes with a local wildcard self-signed `server.crt` certificate that you can find under the `/installation/certs/workspace/raw/` directory of the `kyma` repository. Trust it on the OS level for convenience.
+2. Clone the Kyma repository to your machine using either HTTPS or SSH. Run this command to clone the repository and change your working directory to `kyma`:
+  <div tabs>
+    <details>
+    <summary>
+    HTTPS
+    </summary>
 
-Follow these steps to "always trust" the Kyma certificate on MacOS:
+    ```
+    git clone https://github.com/kyma-project/kyma.git ; cd kyma
+    ```
+    </details>
+    <details>
+    <summary>
+    SSH
+    </summary>
 
-1. Change the working directory to `installation`:
+    ```
+    git clone git@github.com:kyma-project/kyma.git ; cd kyma
+    ```
+    </details>
+  </div tabs>
 
+3. Choose the release from which you want to install Kyma. Go to the GitHub releases page to find out more about each of the available releases. Run this command to list all of the available tags that correspond to releases:
   ```
-  cd installation
+  git tag
   ```
 
-2. Run this command:
-
+4. Checkout the tag corresponding to the Kyma release you want to install. Run:
   ```
-  sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain certs/workspace/raw/server.crt
+  git checkout {TAG}
   ```
+ >**NOTE:** If you don't checkout any of the available tags, your sources match the state of the project's `master` branch at the moment of cloning the repository.
 
->**NOTE:** "Always trusting" the certificate does not work with Mozilla Firefox.
-
-To access the Application Connector and connect an external solution to the local deployment of Kyma, you must add the certificate to the trusted certificate storage of your programming environment. Read [this](/components/application-connector#details-access-the-application-connector-on-a-local-kyma-deployment) document to learn more.
```

Comment:
This is skipped for purpose?

Reply:
I brought it back and made part of the note under the step that shows how to add the certificate to the os keychain.